main.py

```diff
@@ -565,3 +553,31 @@ def handle_stats_reply(self, msg, switch):
             event = KytosEvent('amlight/flow_stats.flows_updated')
             event.content['switch'] = switch.dpid
             self.controller.buffers.app.put(event)
+
+    @listen_to('kytos/of_core.flow_stats.received')
+    def handle_stats_received(self, event):
+        """Capture and handle flow stats messages for OpenFlow 1.3."""
+        msg = event.content['message']
```

Comment:
@gretelliz I'd also encourage you to extract a new method here and call it sometthing like `handle_stats_received` and then this one could be `on_flow_stats_received` or something similar for testability, otherwise it'd run on a Thread and it could lead to non-deterministic behavior on unit test. In the future, we'll include an option in the core to save the decorated mehtod just so it can be tested individually, so that should facilitate in the future. If you need an example check out this part on flow_manager https://github.com/kytos-ng/flow_manager/blob/master/main.py#L214-L222 (notice that the decorated method straight away call anothe rmethod, and that one you can test/cover it more easily)